students/prgrover/lesson04/mailroom.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+from collections import OrderedDict
+
+# Global dictionary containing donor list and donation amounts
+donor_dict = {"Harry Potter": [10000, 5000, 500.55], "Ronald Weasley": [2499.99, 7500.01], "Hermione Granger": [100, 2000, 30000], "Draco Malfoy": [10, 888.88], "Neville Longbottom": [10]}
+
+# Email template for Thank You message
+email = """Dear {:s},
+
+          Thank you for your very kind donation of ${:.2f}.
+          
+          It will be put to very good use.
+                         
+                Sincerely,
+                    -Hogwarts School of Witchcraft and Wizardry
+        """
```

Comment:
Checkout `text wrap.dedent` from the Python standard library for a way to left align the output string while maintaining your current indent level in the code here (so you won't end up with the extra indentation in your generated letters)